### PR TITLE
Fixed #58 Added our first API endpoints

### LINKED ISSUE #58: Add REST API endpoints
Preparations for #7.

### PR DESCRIPTION
**Pull request by FaaPz**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -14,3 +14,22 @@
 $router->get('/', function () use ($router) {
     return $router->app->version();
 });
+
+$router->get('requests', function () use ($router) {
+    return 'get.requests';
+});
+$router->get('requests/{requestId:[0-9]+}', function ($requestId) use ($router) {
+    return "get.requests.{$requestId}";
```
Comment (sjongejan) [inline routes/web.php]: Use single quotes instead of double quotes.

File: routes/web.php [hunk 2]
```diff
@@ -14,3 +14,22 @@
...
+});
+$router->post('requests', function () use ($router) {
+    return 'post.requests';
+});
+
+$router->group(['prefix' => 'requests/{requestId:[0-9]+}'], function ($requestId) use ($router) {
+    $router->get('responses', function ($requestId) use ($router) {
+        return "get.requests.{$requestId}.responses";
```
Comment (sjongejan) [inline routes/web.php]: Use single quotes instead of double quotes.

File: routes/web.php [hunk 3]
```diff
@@ -14,3 +14,22 @@
...
+    });
+    $router->get('responses/{responseId:[0-9]+}', function ($requestId, $responseId) use ($router) {
+        return "get.requests.{$requestId}.responses.{$responseId}";
```
Comment (sjongejan) [inline routes/web.php]: Use single quotes instead of double quotes.

**Review by sjongejan — CHANGES_REQUESTED** [review]

**Comment by FaaPz** [comment]
Committed new changes. Please re-review @sjongejan.

**Review by sjongejan — APPROVED** [review]

**Pull request merged**